src/execution/sql/filter_manager.cpp

```diff
@@ -0,0 +1,133 @@
+#include "execution/sql/filter_manager.h"
+
+#include <memory>
+#include <numeric>
+#include <utility>
+#include <vector>
+
+#include "execution/bandit/agent.h"
+#include "execution/bandit/multi_armed_bandit.h"
+#include "execution/bandit/policy.h"
+#include "execution/sql/projected_columns_iterator.h"
+#include "execution/util/timer.h"
+#include "loggers/execution_logger.h"
+
+namespace terrier::sql {
+
+namespace {
```

Comment:
I appreciate the convenience factor, but how much are we relying on it?  I'm a moderately concerned that it will make the code harder to read (and hence maintain) moving forward.

Reply:
I'm concerned that if we're using it to avoid name conflicts, then we're making it harder to read the code.  Specifically, we do not use it elsewhere in the code so knowing the current namespace is sufficient to know what function a specific line calls without looking anywhere else in a file.  These anonymous namespaces have the property that they could allow masking previous implementations (potentially in the same namespace).